### ISSUE TITLE
Yet Another Wilder Wild Issue

### ISSUE THREAD

**Issue by Nekkok** [Sinytra/Connector#758]
### Mod Name

Wilder Wild

### Mod Homepage

https://modrinth.com/mod/wilder-wild

### Minecraft version

1.20.1

### Describe the bug

Game crashes before loading

### Steps to reproduce

1. Install:
- Forge 47.2.0
- Synitra beta.33
- Forgified Fabric API 0.91.0+1.10.8+1.20.1
- Wilder Wild 2.2.3

2. Run the game
3. Profit

### Logs

https://gist.github.com/Nekkok/8ead311d7be0982d35e0ccf04c1eb0ad

### Additional context

You can find me around your discord n_n

**Comment by murao-kun** [comment]
Can confirm. https://mclo.gs/HJn6TjW

**Comment by uthw** [comment]
I have the same issue, but Wilder Wild 2.1.14 and Forge 47.1.3 work without any problems for me. Newer versions of either crash my game on launch.

[NeoForge 47.1.100 and Wilder Wild 2.1.14](https://mclo.gs/RIEhqAQ)
[Forge 47.1.3 and Wilder Wild 2.2.3](https://mclo.gs/1IlHjv5)
[NeoForge 47.1.100 and Wilder Wild 2.2.3](https://mclo.gs/PCyXiJj)

**Comment by SaltSouls** [comment]
Yeah, also getting this. Seems to be a mixin related issue specifically caused by this:
```
Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Callback method wilderWild$isBubbleColumnOrMesogleaColumn(Lnet/minecraft/world/level/block/state/BlockState;Lnet/minecraft/world/level/block/Block;Lcom/llamalad7/mixinextras/injector/wrapoperation/Operation;)Z in wilderwild.mixins.json:block.mesoglea.LivingEntityMixin from mod wilderwild failed injection check, (0/1) succeeded. Scanned 1 target(s)
```

**Comment by murao-kun** [comment]
Seems to be fixed!

**Comment by SaltSouls** [comment]
> Seems to be fixed!

What is your mods list and the version you're playing on? I just updated the mod recently and am still getting the same set of errors.

**Comment by Su5eD** [comment]
> Seems to be fixed!

Thanks for the update, I tested the mod in a clean instance with Connector beta 42 and it indeed appears to work as expected.

![](https://github.com/Sinytra/Connector/assets/51261569/fee113b5-30a2-4984-b617-c7bd9243aac3)

![](https://github.com/Sinytra/Connector/assets/51261569/b83393ba-aa62-43ce-8321-43674999f63f)

**Comment by GermanArtur** [comment]
https://mclo.gs/k5KEOdt

WilderWild-2.3.3-Fabric+1.20.1.jar
Connector-1.0.0-beta.42+1.20.1.jar
fabric-api-0.92.1+1.11.7+1.20.1.jar
ConnectorExtras-1.10.0+1.20.1.jar

Latest Forge (.23)

**Comment by SaltSouls** [comment]
> > Seems to be fixed!
> 
> Thanks for the update, I tested the mod in a clean instance with Connector beta 42 and it indeed appears to work as expected.
> 
> ![](https://private-user-images.githubusercontent.com/51261569/322486145-fee113b5-30a2-4984-b617-c7bd9243aac3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMyMzE0NjAsIm5iZiI6MTcxMzIzMTE2MCwicGF0aCI6Ii81MTI2MTU2OS8zMjI0ODYxNDUtZmVlMTEzYjUtMzBhMi00OTg0LWI2MTctYzdiZDkyNDNhYWMzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDAxMzI0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY1MjRhMjRmOWU2OTA0OTQ1NjYzZjk2ZjU3NDA3YzhkYzk5MGE4MmMyNmQ1ZjQ3NjU5NmYwOTcxN2MzMGFjNWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.8Lt19XQ4ppjMJBC2B1KkxCevFj9O-tsHgaUDqCvH-aM)
> 
> ![](https://private-user-images.githubusercontent.com/51261569/322486644-b83393ba-aa62-43ce-8321-43674999f63f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMyMzE0NjAsIm5iZiI6MTcxMzIzMTE2MCwicGF0aCI6Ii81MTI2MTU2OS8zMjI0ODY2NDQtYjgzMzkzYmEtYWE2Mi00M2NlLTgzMjEtNDM2NzQ5OTlmNjNmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDAxMzI0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMzNDBiYjNmNTRhZTE3ZWUzMDQ2YTU2MWUzNTQwMTgyMTZjNDNkZTkzNmI5NmM1ZDM1YTU2ZWU0ZGQzNGU3OTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.2f7VdpZiwt1gNtC_GKeA8EtLvWybxDiCLAURG8wCpGo)

So I hate to necro this as it is already fixed and I technically got it working on my end as well, but there is an issue with it and embeddium. In order to fix this issue, you need to go into mixins.json5 and under `only aplies if sodium is installed...` disable `block_mesoglea`. After which it will launch properly. So it would seem the mixin problem is still there, but only if you have something like rubidium or embeddium installed.

**Comment by PulcioPappi** [comment]
> So I hate to necro this as it is already fixed and I technically got it working on my end as well, but there is an issue with it and embeddium. In order to fix this issue, you need to go into mixins.json5 and under `Only applies if Sodium is installed...` disable `block_mesoglea`. After which it will launch properly. So it would seem the mixin problem is still there, but only if you have something like rubidium or embeddium installed.

Where is this file located? I can't find it
